agent/src/agent/pipeline/config/stages/base.py

```diff
@@ -31,10 +30,10 @@ def _get_script(self) -> str:
             return f.read()
 
 
-class JythonScript(Stage, ABC):
+class _JythonScript(Stage, ABC):
```

Comment:
This class is kinda internal, JythonSource and JythonProcessor inherit from it and only they should be used as there are only 2 types of Jython scripts

Reply:
It's an abstract class anyway, no one will be able to use it and if you will want to inherit from it outside this module it will be weird with the underscore

Reply:
yes, this is the plan) I don't want anyone to inherit from it outside this module